bin/grommet.js

```diff
@@ -43,8 +43,8 @@ var tasks = {
         process.chdir('./' + app);
 
         var templateFolder = path.join(grommetPath, 'templates/init/**');
-        var mobileIcon = path.join(grommetPath, 'mobile-app-icon.png');
-        var shortcutIcon = path.join(grommetPath, 'shortcut-icon.png');
+        var mobileIcon = path.join(grommetPath, 'src/img/mobile-app-icon.png');
```

Comment:
I believe this should be `img/mobile-app-icon.png`. When we install grommet, there is no `src` anymore.